### PR TITLE
refactor(editor): Type node view event bus (no-changelog)

### PR DESCRIPTION
**Pull request by tomi**
## Summary

Builds on top of #10367

## Related Linear tickets, Github issues, and Community forum posts

https://linear.app/n8n/issue/CAT-32/type-event-bus-usages-in-editor

## Review / Merge checklist

- [ ] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/editor-ui/src/event-bus/node-view.ts [hunk 1]
```diff
@@ -1,3 +1,30 @@
...
+
+export interface NodeViewEventBusEvents {
+	/** Command to create a new workflow */
+	newWorkflow: never;
+
+	/** Command to open the chat */
+	openChat: never;
+
+	/** Command to save the current workflow */
+	saveWorkflow: CallbackFn;
```
Comment (netroy) [inline packages/editor-ui/src/event-bus/node-view.ts:15]: using this particula alias feels confusing. We should either call the type `OnSaveWorkflowFn` or replace the line with `saveWorkflow: () => void;`

**Review by netroy — CHANGES_REQUESTED** [review]

File: packages/editor-ui/src/event-bus/node-view.ts (hunk 1 above)
Comment (tomi) [reply packages/editor-ui/src/event-bus/node-view.ts:15]: Agreed. I tried to clarify it

**Review by netroy — APPROVED** [review]

**Comment by netroy** [comment]
no point blocking this PR on the flaky tests. I'll merge this.

**Comment by janober** [comment]
Got released with [`n8n@1.56.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.56.0)

**Pull request merged**